src/resolve/resolved-mdns.c

```diff
@@ -265,6 +265,7 @@ static int on_mdns_packet(sd_event_source *s, int fd, uint32_t revents, void *us
 
```

Comment:
Not a matter, as we usually ignore typo in commit message, `we'd pas` -> `we'd pass`